src/components/titles/index.js

```diff
@@ -160,12 +229,41 @@ function draw(gd, titleClass, options) {
             lineposition: fontLineposition,
         })
         .attr(attributes)
-        .call(svgTextUtils.convertToTspans, gd);
+            .call(svgTextUtils.convertToTspans, gd, adjustSubtitlePosition);
+
+        if(subtitleEl) {
+            // Increase the subtitle y position so that it is drawn below the subtitle
+            // We need to check the height of the MathJax group as well, in case the MathJax
+            // has already rendered
+            var titleElHeight = titleEl.node().getBBox().height;
+            var titleElMathGroup = group.select('.' + titleClass + '-math-group');
+            var titleElMathHeight = titleElMathGroup.node() ? titleElMathGroup.node().getBBox().height : 0;
+            var subtitleShift = titleElMathHeight ? titleElMathHeight + MATHJAX_PADDING_BOTTOM : titleElHeight;
+            var subtitleAttributes = Object.assign({}, attributes);
+            subtitleAttributes.y += subtitleShift;
```

Comment:
`Object.assign` is not used in many places in our code.
You may consider rewriting this using ` Lib.extendFlat` like this: 
```suggestion
            var subtitleAttributes = Lib.extendFlat({}, attributes, {
              y: subtitleShift + attributes.y
            });
```